docs/explain.md

```diff
@@ -0,0 +1,46 @@
+# Explain
+
+When reviewing inferred axioms, such as after applying the [`reason`](/reason) command,
+you may like more information about why a particular axiom was inferred:
+
+    robot explain -i uvula.ofn -r ELK --axiom "'uvular muscle' SubClassOf 'muscle organ' and 'part of' some 'jaw region'" -o uvular_muscle.md
```

Comment:
It would look better on the doc page if this was split as so:
```
robot explain --input uvula.ofn --reasoner ELK \
  --axiom "'uvular muscle' SubClassOf 'muscle organ' and 'part of' some 'jaw region'" \
  --output uvular_muscle.md
```

Would it be beneficial to also have an `--axiom-file` input option to provide a text file for the axiom? Multiple lines for multiple axioms that you'd like to explain?

Reply:
Re: `--axiom-file`, I would be fine with this, but don't have a need for it now. We could always add it down the road in a backward-compatible way.